src/hb-buffer-serialize.cc

```diff
@@ -196,6 +196,47 @@ _hb_buffer_serialize_glyphs_json (hb_buffer_t *buffer,
   return end - start;
 }
 
+static unsigned int
+_hb_buffer_serialize_unicode_json (hb_buffer_t *buffer,
+					unsigned int start,
+					unsigned int end,
+					char *buf,
+					unsigned int buf_size,
+					unsigned int *buf_consumed)
+{
+	hb_glyph_info_t *info = hb_buffer_get_glyph_infos (buffer, nullptr);
```

Comment:
This is because in the .editorconfig, there is the following line:

```
indent_style = tab # should be space
```

Maybe that should be "space"? :-)

Reply:
I have no idea, I don’t use an editor that understands this config file.